cmd/podman/rm.go

```diff
@@ -61,6 +90,31 @@ func rmCmd(c *cliconfig.RmValues) error {
 	}
 	defer runtime.Shutdown(false)
 
+	if rootless.IsRootless() {
+		var container *libpod.Container
+		if c.All || len(c.InputArgs) > 1 {
+			return errors.New("rootless mode doesn't support more than 1 container")
```

Comment:
Please add a note about this in the man page.

Reply:
done